src/cephadm/cephadm.py

```diff
@@ -3386,6 +3526,10 @@ def get_container(ctx: CephadmContext,
         name = '%s.%s' % (daemon_type, daemon_id)
         envs.extend(Keepalived.get_container_envs())
         container_args.extend(['--cap-add=NET_ADMIN', '--cap-add=NET_RAW'])
+    elif daemon_type == CephNvmeof.daemon_type:
+        name = '%s.%s' % (daemon_type, daemon_id)
+        container_args.extend(['--ulimit', 'memlock=-1:-1'])  # TODO: FIXME
```

Comment:
Also `--ulimit nofile` should be set. Right now to a higher value (e.g.: ten times the default, 10240), but it might make sense to set it unlimited too.

Reply:
@epuertat I'll use `10240` for the moment, we can increase the value in case we find the current limit is not reasonable.

Reply:
@epuertat By default I'll set it to `10240` but this option can be configured by the feature of [extra container args](https://docs.ceph.com/en/latest/cephadm/services/#extra-container-arguments) so users can updated to fit their needs.